groups/instance/cloud-init/templates/enable_config.tpl

```diff
@@ -808,5 +764,8 @@ write_files:
       ${artifactory_license}
 
 runcmd:
-  - systemctl enable artifactory 
-  - systemctl restart artifactory
\ No newline at end of file
+  - systemctl enable artifactory
+  - sudo mkdir /var/lib/artifactory
+  - sudo mount -t efs ${efs_dns_name}:/ /var/lib/artifactory
+  - sudo echo "${efs_dns_name}:/ /var/lib/artifactory efs defaults,_netdev 0 0" >> /etc/fstab
```

Comment:
May be worth reviewing the mount options here to ensure we're setting sensible defaults for NFS/EFS

E.g:
https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html

Reply:
Hi Wayne, I have crossed checked the logs of what values are being implemented from the mounting commands where these match the defaulted recommendations. Although ....

I did notice that I was not using TLS with either of the EFS mounting commands.

To enable TLS, I have changed the EFS system policy to set the ‘secureTransport’ to true. From there I have added the ‘tls’ option to both mount commands. This also results in the efs-mount-watchdog being enabled & is monitoring the TLS tunnel config for the EFS & Instance. 

Tested with a new deployment, reboot & stop / start process of the instance. EFS continues to connect with read & write abilities remaining.